dcpy/connectors/edm/recipes.py

```diff
@@ -248,3 +305,62 @@ def purge_recipe_cache():
     """Delete locally stored recipe files."""
     logger.info(f"Purging local recipes from {LIBRARY_DEFAULT_PATH}")
     shutil.rmtree(LIBRARY_DEFAULT_PATH)
+
+
+def scrape_metadata(dataset: str) -> None:
+    """For a given recipes dataset, regenerates logs"""
+    logger.info(f"Re-scraping metadata for dataset {dataset}")
+    pg_client = postgres.PostgresClient(database=LOGGING_DB, schema=LOGGING_SCHEMA)
+
+    ## remove records in db for this recipe dataset
+    try:
+        pg_client.execute_query(
+            f"""
+            DELETE FROM {LOGGING_TABLE_NAME}
+            WHERE name = :dataset AND event_source='scrape';""",
+            dataset=dataset,
+        )
+    except Exception as e:
+        logger.warn(f"Could not delete from table: {e}")
+
+    ## scrape metadata, insert into table
+    versions = get_all_versions(dataset)
+    metadata = [
+        get_metadata(dataset, v).to_row()
+        for v in versions
+        #    if v not in ["testor", "2023_executive_points", "2017_adopted_points", "2017_adopted_polygons"]
+        #    and not (dataset == "doitt_buildingfootprints" and v == "20230414")
+        #    and not (dataset == "test_nypl_libraries")
+    ]
+    metadata_df = pd.DataFrame.from_records(metadata)
+    metadata_df["event_source"] = "scrape"
+    pg_client.insert_dataframe(
```

Comment:
As you can see, went a very lightweight approach for now of "migrations" - just insert df, append to existing table. So pandas.to_sql creates table if it doesn't exist